ui/Rhino/AGS/dev/AGS_force_location_cmd.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+
+import compas_rhino
+
+from compas.geometry import subtract_vectors
+from compas.geometry import add_vectors
+
+
+__commandname__ = "AGS_force_location"
+
+
+def RunCommand(is_interactive):
+    if 'AGS' not in sc.sticky:
+        compas_rhino.display_message('AGS has not been initialised yet.')
+        return
+
+    scene = sc.sticky['AGS']['scene']
+    force = scene.find_by_name('Force')[0]
+
+    start = compas_rhino.pick_point('Pick a point to move from.')
+    if start:
+        end = compas_rhino.pick_point('Pick a point to move to.')
+        if end:
+            vector = subtract_vectors(end, start)
+            xyz = force.artist.anchor_point
+            new_xyz = add_vectors(xyz, vector)
+            force.artist.anchor_point = new_xyz
```

Comment:
Can you rewrite this with `move_vertices(...)`? This would be nicer since the shadow of the diagram would move together with the mouse, giving the right impression of what's happening.

Reply:
ummm... could we leave this enhancement in the future? I would add it to the todo list. (think there would be more bugs to solve first... )